### PR TITLE
AllowSingleDeselect tests

### PR DESCRIPTION
**Pull request by spg**

### CODE REVIEW

**Comment by christiangoudreau** [comment]
LGTM

File: integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/testcases/AllowSingleDeselect.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        chosenOptions.setPlaceholderText(PLACEHOLDER);
+        chosenOptions.setAllowSingleDeselect(true);
+
+        ChosenValueListBox<CarBrand> listBox = new ChosenValueListBox<CarBrand>(RENDERER, chosenOptions);
+
+        ArrayList<CarBrand> acceptableValues = Lists.newArrayList(CarBrand.values());
+        acceptableValues.add(0, null);
+        listBox.setAcceptableValues(acceptableValues);
+
+        RootPanel.get().add(listBox);
```
Comment (meriouma) [inline integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/testcases/AllowSingleDeselect.java]: What does this test verify?

File: integration-test/src/test/java/SampleIT.java [hunk 2]
```diff
@@ -77,11 +80,26 @@ public void showNonEmptyValues() {
         assertThat(options).isEqualTo(allNames);
     }
 
+    @Test
+    public void allowSingleDeselect() {
+        loadTestCase(new AllowSingleDeselect());
```
Comment (meriouma) [inline integration-test/src/test/java/SampleIT.java]: Ah, you juste use the TestCase as a kind of setup right?

**Comment by meriouma** [comment]
LGTM

**Comment by jDramaix** [comment]
LGTM

File: integration-test/src/test/java/SampleIT.java (hunk 2 above)
Comment (spg) [inline integration-test/src/test/java/SampleIT.java]: @merioumas exactly

**Pull request merged**